build-tools/packages/version-tools/src/versions.ts

```diff
@@ -10,7 +10,7 @@ import { detectVersionScheme, getLatestReleaseFromList } from "./schemes";
 import { changePreReleaseIdentifier, isInternalVersionScheme } from "./internalVersionScheme";
 
 // TODO: Replace this with a shared release group type.
-type TagPrefix = string | "client" | "server" | "azure" | "build-tools";
+export type TagPrefix = string | "client" | "server" | "azure" | "build-tools";
```

Comment:
Same with this. These are artifacts of the current dep direction and aren't intended to be exported because they're duplicative.